Parse/Internal/Query/Utilities/PFQueryUtilities.m

```diff
@@ -86,17 +86,11 @@ + (NSPredicate *)_negatePredicate:(NSPredicate *)predicate {
                              }
                              return [NSCompoundPredicate andPredicateWithSubpredicates:newSubpredicates];
                          }
-                         default: {
```

Comment:
I still like having default use cases here. It protects us from some stupid person coming along with a custom predicate and destroying anything. I know it's less optimal, but silent failures are bad.